js/modules/k6/k6.go

```diff
@@ -161,9 +162,11 @@ func (*K6) Check(ctx context.Context, arg0, checks goja.Value, extras ...goja.Va
 		if ok {
 			tmpVal, err := fn(goja.Undefined(), arg0)
 			if err != nil {
-				return false, err
```

Comment:
I would rather rewrite to:

```
tmpVal, err := fn(goja.Undefined(), arg0)
val = tmpVal
if err != nil {
                val = rt.ToValue(false)
                exc = err
}
```

Reply:
Thanks, it avoids a branch, not sure how I missed it. Changed.